In [3]:
import warnings
warnings.simplefilter(action='ignore', category=FutureWarning)

from report_agent_Tool import ReportAgentTool

tool = ReportAgentTool()
result = tool.invoke({"query": "삼성전자의 주가를 예측해줘", 
             "metadata": {"companyName": "삼성전자"}})
print("\n\n"+"="*100+"\n\n")
print(result)


=== 리포트 에이전트 프로세스 시작 ===
입력 쿼리: 삼성전자의 주가를 예측해줘
필터 조건: {'companyName': '삼성전자'}
회사명: 삼성전자
티커: 005930
===1. 회사명 및 티커 추출 완료===
===2. 분기별 요약손익계산서 및 사업부별 매출액 추출 완료===
===3. 직전분기 매출액 컨센서스 추출 완료===
===4. 직전분기 누적 매출액 컨센서스 및 yoy 컨센서스 계산 완료===
===5. 사업부별 YOY 계산 완료===
===6. 사업부별 매출액(영업수익) 추출 완료===
===7. 직전분기 실적 리뷰 작성 완료===
===8. 사업부별 뉴스 검색 완료===
===9. 사업부별 YOY 예측 완료===
===10. 다음분기 사업부별 매출액 yoy 예측 결과와 근거 작성 완료===
===11. 다음 분기 매출액, 영업이익, 순이익 예측 완료===
{'SK Hynix': 12.511711475154545, 'Micron Technology': 18.045421165996906, 'Qualcomm': 12.156259684007098}
===12. 목표 PER 산정을 위한 peer PER 및 현재 PER 확인 완료===
===13. 밸류에이션(목표 주가 산정) 완료===
===14. 리포트 최종 출력 완료===
삼성전자 리포트

직전 분기 실적 리뷰:

이번 분기 실적을 분석한 결과, 총 매출액은 225조 8263억 4000만 원으로 전년 동기 대비 17.75% 증가하였습니다. 이는 시장 컨센서스인 226조 8841억 5300만 원에 약간 못 미치는 결과로, 컨센서스 대비 0.86% 낮은 수치를 기록하였습니다.

사업부별로 살펴보면, DX 사업부는 134조 3575억 원의 매출을 기록하며 전년 동기 대비 3.00% 증가하였습니다. 이는 안정적인 성장세를 유지하고 있음을 보여줍니다. 반면, DS 사업부는 80조 9652억 원의 매출로 전년 동기 대비 80.31%라는 큰 폭의 성장을 기록하였습니다. 이는 DS 사업부의 강력한 실적 개

'삼성전자 리포트\n================\n\n직전 분기 실적 리뷰:\n\n이번 분기 실적을 분석한 결과, 총 매출액은 225조 8,263억 4,000만 원으로 전년 동기 대비 17.75% 증가하였습니다. 이는 시장 컨센서스인 226조 8,415억 3,000만 원에 약간 못 미치는 결과로, 컨센서스 대비 0.94% 낮은 수치를 기록하였습니다.\n\n사업부별로 살펴보면, DX 사업부는 134조 3,575억 원의 매출을 기록하며 전년 동기 대비 3.00% 증가하였습니다. 이는 안정적인 성장세를 유지하고 있음을 보여줍니다. 반면, DS 사업부는 80조 9,652억 원의 매출로 전년 동기 대비 80.31%라는 큰 폭의 성장을 기록하였습니다. 이는 반도체 수요 증가와 관련된 긍정적인 시장 환경이 반영된 결과로 보입니다.\n\nSDC 사업부는 21조 317억 원의 매출을 기록하며 전년 동기 대비 1.33% 감소하였습니다. 이는 디스플레이 시장의 경쟁 심화와 수요 감소가 영향을 미친 것으로 분석됩니다. Harman 사업부는 10조 3,489억 원의 매출로 전년 동기 대비 1.10% 감소하였으며, 이는 글로벌 자동차 시장의 불확실성과 관련된 요인들이 작용한 것으로 판단됩니다.\n\n종합적으로, 이번 분기 실적은 DS 사업부의 강력한 성장세가 전체 매출 증가를 견인하였으나, SDC와 Harman 사업부의 매출 감소가 일부 상쇄된 결과로 볼 수 있습니다. 향후 시장 환경 변화에 대한 지속적인 모니터링과 사업부별 전략 조정이 필요할 것으로 보입니다.\n\n--------------------------------------------------\n\n### DX 사업부\n- **예측 매출액 변화율**: 다음 분기 DX 사업부의 매출액은 yoy로 2.5% 변화할 것으로 예측됩니다.\n- **직전 분기 매출액 변화율**: 3.000059029116687% 변화했습니다.\n- **근거**: 최근 뉴스에 따르면, DX 사업부 내에서 반도체 사업부와 비교하여 보너스 분배의 불균형으로 인한 내부 불만이 제기되었습니다. 이는 직원들의 사기와 생산성에 부정적인 영향을 미칠 수 있으며, 이에 따라 예상 성장률이 약간 감소할 수 있습니다. 또한, 안정성에 중점을 두고 운영 이익을 개선하는 데 어려움을 겪고 있는 상황이 보수적인 전망을 제시합니다.\n\n### DS 사업부\n- **예측 매출액 변화율**: 다음 분기 DS 사업부의 매출액은 yoy로 20.0% 변화할 것으로 예측됩니다.\n- **직전 분기 매출액 변화율**: 80.31495186193965% 변화했습니다.\n- **근거**: DS 사업부의 영업이익률(OPI)이 작년 0%에서 올해 12-16%로 증가하여 이전의 침체에서 회복되고 있음을 나타냅니다. 이는 재무 성과의 개선을 시사하지만, 여전히 평균 수준에는 미치지 못하고 있습니다. 이전의 yoy 80.31%를 고려할 때, 지속적인 회복이 진행 중이지만 아직 최고 성과에 도달하지 못했기 때문에 보다 완만한 성장률인 20%가 예상됩니다.\n\n### SDC 사업부\n- **예측 매출액 변화율**: 다음 분기 SDC 사업부의 매출액은 yoy로 0.05% 변화할 것으로 예측됩니다.\n- **직전 분기 매출액 변화율**: -1.3328141566349845% 변화했습니다.\n- **근거**: SDC 사업부의 매출은 전 분기 대비 5% 증가하여 8조 원에 도달했습니다. 이러한 긍정적인 성장은 전 분기의 감소와 비교하여 연간 실적이 개선될 가능성을 시사합니다. 매출 증가가 회복 또는 성장 추세를 나타내며, 이는 연간 변화율의 긍정적인 변화를 이끌 수 있습니다.\n\n### Harman 사업부\n- **예측 매출액 변화율**: 다음 분기 Harman 사업부의 매출액은 yoy로 -1.05% 변화할 것으로 예측됩니다.\n- **직전 분기 매출액 변화율**: -1.1009069102932885% 변화했습니다.\n- **근거**: 삼성의 반도체 사업부가 직면한 전반적인 어려움에도 불구하고, MX 및 네트워크 사업부의 강력한 성과는 모바일 및 네트워크 솔루션에 대한 통합 기술 수요 증가를 통해 Harman의 매출에 긍정적인 영향을 미칠 수 있습니다. 그러나 반도체 부문의 어려움이 지속되고 있는 상황에서 경쟁과 시장 압력이 계속되고 있어 전 분기의 yoy 감소에서 약간의 개선만 예상됩니다.\n\n이 예측은 다음과 같은 뉴스를 기반으로 이루어졌습니다:\n- [삼성전자 버팀목 된 MX사업부…AI폰 플래그십 전략 먹혔다](https://www.msn.com/ko-kr/news/other/삼성전자-버팀목-된-mx사업부-ai폰-플래그십-전략-먹혔다/ar-AA1tfkKy)\n- [삼성전자, 3Q 매출 79.1조원·영업익 9.2조원…"4Q 실적 개선 전망"](https://www.ebn.co.kr/news/articleView.html?idxno=1641822)\n- [삼성전자, TSMC와 엎치락뒤치락… 3분기 반도체 매출 1위 내줄듯](https://www.msn.com/ko-kr/news/other/삼성전자-tsmc와-엎치락뒤치락-3분기-반도체-매출-1위-내줄듯/ar-AA1sBlSA)\n\n--------------------------------------------------\n\n삼성전자의 목표 주가를 설정하기 위해, 먼저 사업부별 매출 실적 리뷰와 예측을 분석하고, 이를 바탕으로 목표 PER을 설정한 후 추정 EPS와 곱하여 목표 주가를 산출하겠습니다.\n\n### 직전 분기 매출 실적 리뷰\n- **총 매출액**: 225조 8,263억 4,000만 원으로 전년 동기 대비 17.75% 증가. 시장 컨센서스 대비 0.94% 낮음.\n- **DX 사업부**: 134조 3,575억 원, 전년 동기 대비 3.00% 증가. 안정적인 성장세 유지.\n- **DS 사업부**: 80조 9,652억 원, 전년 동기 대비 80.31% 증가. 반도체 수요 증가에 따른 긍정적 시장 환경 반영.\n- **SDC 사업부**: 21조 317억 원, 전년 동기 대비 1.33% 감소. 디스플레이 시장의 경쟁 심화와 수요 감소 영향.\n- **Harman 사업부**: 10조 3,489억 원, 전년 동기 대비 1.10% 감소. 글로벌 자동차 시장의 불확실성 영향.\n\n### 사업부별 매출 예측\n- **DX 사업부**: 다음 분기 매출액 yoy 2.5% 증가 예상. 내부 불만과 운영 이익 개선의 어려움으로 보수적 전망.\n- **DS 사업부**: 다음 분기 매출액 yoy 20.0% 증가 예상. 영업이익률 개선으로 회복세 지속.\n- **SDC 사업부**: 다음 분기 매출액 yoy 0.05% 증가 예상. 전 분기 대비 매출 증가로 연간 실적 개선 가능성.\n- **Harman 사업부**: 다음 분기 매출액 yoy -1.05% 감소 예상. 반도체 부문 어려움 지속으로 약간의 개선만 예상.\n\n### 목표 PER 설정\n현재 삼성전자의 PER은 11.17이며, 경쟁사들의 평균 PER은 17.21입니다. 삼성전자는 DS 사업부의 강력한 성장세와 DX 사업부의 안정적인 성장을 바탕으로 긍정적인 실적을 보이고 있습니다. 그러나 SDC와 Harman 사업부의 매출 감소는 일부 리스크로 작용할 수 있습니다. 이를 고려하여 목표 PER을 경쟁사 평균보다 낮은 15.0으로 설정하겠습니다.\n\n### 목표 주가 계산\n- **추정 EPS**: 5948.26\n- **목표 PER**: 15.0\n- **목표 주가**: 5948.26 * 15.0 = 89,223.90 원\n\n### 목표 주가 컨센서스\n- **평균 목표 주가**: 80,791.43 원\n- **최소 목표 주가**: 48,000.00 원\n- **최대 목표 주가**: 125,000.00 원\n\n삼성전자의 목표 주가는 89,223.90 원으로, 이는 평균 목표 주가인 80,791.43 원보다 높고, 최대 목표 주가인 125,000.00 원보다는 낮은 수준입니다. 이는 DS 사업부의 강력한 성장세와 DX 사업부의 안정적인 성장을 반영한 결과입니다. 향후 시장 환경 변화와 사업부별 전략 조정에 따라 목표 주가의 변동 가능성이 있으므로 지속적인 모니터링이 필요합니다.\n\n--------------------------------------------------\n\n\n본 자료는 사업보고서, 기사 등 신뢰할 수 있는 자료를 바탕으로 작성되었습니다.\n그러나, AI의 의견이 반영되었으며 정확성이나 완전성을 보장할 수 없습니다.\n본 자료는 참고용으로 투자자 자신의 판단과 책임하에 투자하시기 바랍니다.\n최종 리포트 작성일 : 2024-12-27'